tasks/task01/narcissistic_numbers.py

```diff
@@ -0,0 +1,19 @@
+def countDigits(n):
+    count = 0
+    while n != 0:
+        n //= 10
+        count += 1
+    return count
+
+
+narcissistic_numbers = []
+for i in range(1, 1001):
+    number_of_digits = countDigits(i)
+    digits = [int(x) for x in str(i)]
```

Comment:
It isn't allowed in the task description.  
Note: Use pure math to find the number of digits in a number. I.e. not via string cast.

You can use math and it could be simple